### ISSUE TITLE
Update syntax and accessibility issues

### ISSUE THREAD

**Issue by rogeruiz** [18F/vote-2016#8]
@carodew brought up some great points on accessibility for the current site. I think we can fix some of the lowest hanging fruit for the single page ( `index.html` ) that shouldn't take more than a day to integrate. When we have these changes in, _we should consider merging it into the original repo and deploying it_.

Some of the issues proposed fixing are around

- `aria-roles` in the modal
- invalid attributes in HTML
- multiple ids in the HTML

@msecret Feel free to add any more to this list!

**Comment by aardrian** [comment]
The biggest issue on the site is the keyboard navigation. Try tabbing into the select menu and then using it with just a keyboard — the arrow keys are a problem since any change fires the fake modal (onchange), as does typing the letter for a state (which in the case of N is never my state on first type). Look what happens when you hit N repeatedly to get to New Jersey. Note how the modal of info is block by the other modals of info?

In addition, the page over-uses headers (and has too many <h1>s), has useless image alt text on the images under "Ways to Register," lacks link text for your social media icons, has useless link text and too-low contrast for the link after "7 in 10," and useless link text for the very last link on the page ("here").

The modal itself is completely useless with a screen reader. The modal is not announced to the screen readers user in any fashion, and is only added at the very bottom of the page. When I finally navigate to it, the "close" button (which it isn't, since it's just a clickable <div>, when it really should be a <button>) is announced as "X."

It feels like this was never tested with just a keyboard nor with a screen reader.

While the 18F Accessibility Guide is still in need of updates (such as a section explicitly dealing with modals), following it would have at least guided your keyboard support: https://pages.18f.gov/accessibility/index.html

**Comment by carodew** [comment]
@aardrian spot on. We are currently working on a design that fixes these issues and removes the modal altogether. We'll post an update on this thread when the changes are live.

**Comment by aardrian** [comment]
@carodew Awesome. Do you have some sense of a timeframe?

**Comment by carodew** [comment]
No definite timeframe, but hopefully sometime in the next few weeks.
